esp/eclwatch/ws_XSLT/services.xslt

```diff
@@ -234,6 +234,16 @@
                 var url = protocol + "://" + window.location.hostname + ":" + port;
                 parent.document.location.href = url;
             }
+            function showPopup(protocol, ip, port)
+            {
+                var url = protocol + "://" + ip + ":" + port;
```

Comment:
can we default to http in case protocol is empty?

Reply:
Will do.